### PR TITLE
Escape all the things

### PR DESCRIPTION
**Pull request by clayton-shopify**
Currently qless doesn't perform any output escaping, which means that cross-site scripting vulnerabilities are very likely present.

To fix this, I've done the following:

* switch the ERB template engine to erubi and enable automatic HTML escaping
* change `<%= erb ... %>` to `<%== erb ... %>` everywhere, so that templates are not double-escaped
* use `to_json` to convert Ruby strings to Javascript strings in all Javascript interpolations

At this point, every interpolation should either be HTML-escaped or Javascript-escaped, depending on the context.

I've done some basic testing to make sure things are still working, but would appreciate a :tophat: from someone more familiar with qless.

### CODE REVIEW

File: lib/qless/server.rb [hunk 1]
```diff
@@ -17,6 +18,8 @@ class Server < Sinatra::Base
     # I'm not sure what this option is -- I'll look it up later
```
Comment (EiNSTeiN-) [inline lib/qless/server.rb]: You can probably remove this then? 😄

Comment (clayton-shopify) [reply lib/qless/server.rb]: That was in reference to `set :static, true`, I think.

Comment (EiNSTeiN-) [reply lib/qless/server.rb]: Yes, those two lines can be removed now?

Comment (clayton-shopify) [reply lib/qless/server.rb]: I guess if nobody looked it up in the last 6 years it's not likely to happen. :smile:

**Comment by clayton-shopify** [comment]
@bslobodin The problem is that I was HTML-escaping inside `<script>` tags. I've switched to `<%== ... %>` for all interpolations that occur inside `<script>` tags, which should resolve the issue.

File: lib/qless/server/views/layout.erb [hunk 2]
```diff
@@ -91,7 +91,7 @@
      * flash a message to that effect on the page */
     var move = function(jid, queue, cb) {
       _ajax({
-        url: '<%= u "/move" %>',
+        url: <%== (u "/move").to_json %>,
```
Comment (Andreis13) [inline lib/qless/server/views/layout.erb]: that's a weird way to call a method 😄 
`u("/move").to_json` vs `(u "/move").to_json` I guess neither of them are very readable...

Comment (clayton-shopify) [reply lib/qless/server/views/layout.erb]: I think `u("/mode").to_json` is better; I'll change it to that.

**Pull request merged**